accord-core/src/test/java/accord/utils/RandomSourceTest.java

```diff
@@ -111,18 +104,101 @@ private double testOneBiasedLongs(RandomSource random, int min, int median, int
             results[i] = random.nextBiasedInt(min, median, max);
 
         Arrays.sort(results);
-        int i = Arrays.binarySearch(results, median);
+        int i = firstBinarySearch(results, median);
         if (i < 0) i = -1 - i;
-        int j = Arrays.binarySearch(results, median + 1);
+        int j = firstBinarySearch(results, median + 1);
```

Comment:
If we're copying `ceil`, we could just use `ceil` directly?

Reply:
renamed the method to `ceil`. I also simplified it to call the forked `ceil` and only exists to simplify the signature.